web_src/less/_repository.less

```diff
@@ -2177,9 +2177,11 @@
 
             > .item {
                 float: left;
-                padding: .5rem .5rem !important;
```

Comment:
Reason：The second one is repeated

Reply:
I've opened https://github.com/go-gitea/gitea/pull/11436 for this, you can remove the change here.